test-requirements.txt

```diff
@@ -1,3 +1,3 @@
 PyVirtualDisplay==0.1.5
-selenium==2.49.0
+selenium==2.49.2
```

Comment:
I'm seeing failures like "TypeError: string indices must be integers", which (from my reading of https://github.com/SeleniumHQ/selenium/issues/1497) appears to have been introduced in 2.49.1, is present in 2.50, and was fixed only three days ago.

On the `develop` branch I see "No JSON object could be decoded" errors, which (from my reading of https://github.com/SeleniumHQ/selenium/issues/1478) were introduced in 2.49.0

All of which begs the question of why the UI tests succeed for you.